java/src/main/java/ai/rapids/cudf/HostMemoryBuffer.java

```diff
@@ -246,7 +246,7 @@ public final void copyFromHostBuffer(long destOffset, HostMemoryBuffer srcData,
    * @param in input stream to copy bytes from
    * @param byteLength number of bytes to copy
    */
```

Comment:
Please update the docs so that it is clear that an EOFException is thrown if the InputStream does not include enough bytes to do the copy. Also it would really be nice if we could include a message with the EOFException to indicate what happened.